src/ucp/api/ucp.h

```diff
@@ -1243,6 +1340,43 @@ ucs_status_t ucp_worker_signal(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Accept connections on a local address of the worker object.
+ *
+ * This routine binds the worker object to a sock_addr which is set by the user.
+ * The worker will listen to incoming connection requests and upon receiving such
+ * a request from the remote peer, an endpoint to it will be created.
+ * The user's call-back will be invoked once the endpoint is created.
+ *
+ * @param [in]  worker           Worker object that is associated with the
+ *                               params object.
+ * @param [in]  params           User defined @ref ucp_worker_listener_params_t
+ *                               configurations for the @ref ucp_listener_h.
+ * @param [out] listener         A handler to the listener, to be used later for
```

Comment:
the goal of the listener is not to be destroyed :)
"A handler to the listener. Can be released by calling @ref ucp_listener_destroy"